### PR TITLE
add LOJ 1010 

### PR DESCRIPTION
**Pull request by Perdente**

### CODE REVIEW

File: 1010/en.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Chess board knights only travel in L shaped squares.
```
Comment (faiyaz26) [inline 1010/en.md:1]: Add problem title like this: https://github.com/lightoj-dev/problem-tutorials/pull/11/files#diff-49d89fcefb3733849902cae167206bfef3d2d2fa1569a2abcc5b303e53f828b9R1

File: 1010/en.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Chess board knights only travel in L shaped squares. 
+
```
Comment (faiyaz26) [inline 1010/en.md:2]: Describe what the problem wants in two three lines, and easier way.

File: 1010/en.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+In this  problem there are 3 cases mainly considerable.
+
+- **Case 1:**  If any of  the rows or columns is 1 then there is no L shaped formed. So answer is max of row and column.[See the pic](https://ibb.co/zF8114s) for better understanding.
```
Comment (faiyaz26) [inline 1010/en.md:5]: Add the image directly, with  tag

File: 1010/en.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+- **Case 2:**  If rows and columns are greater then 2 then just half the total of squares is the answer.
+
+- **Case 3:**(most important case) If either of the row or column is 2 then we just divide the total squares by 8 where 1st 4 blocks can be placed by knights and other 4 blocks are forbidden. [See the pic](https://ibb.co/sJPy0Vr) for better understanding.
+The remaining blocks(if any) if less then or equals 4 then remaining  blocks are added to the answer and if greater then we add 4 to the answer.
+
+```
```
Comment (faiyaz26) [inline 1010/en.md:12]: add the full code,

use the language syntax with markdown, for example:

```c

**Review by faiyaz26 — CHANGES_REQUESTED** [review]

File: 1010/en.md (hunk 4 above)
Comment (faiyaz26) [reply 1010/en.md:12]: Add language header like this: https://github.com/lightoj-dev/problem-tutorials/pull/11/files#diff-49d89fcefb3733849902cae167206bfef3d2d2fa1569a2abcc5b303e53f828b9R13-R15

Comment (Perdente) [reply 1010/en.md]: I have updated my tutorial.You should check this out

**Review by Perdente — COMMENTED** [review]
check this if something is missing

**Pull request merged**